File: 백준/Bronze/2562. 최댓값/최댓값.c
#include <stdio.h>

int main() {
	int input;
	int max = 0;
	int num = 0;

	for (int i = 1; i < 10; i++) {
		scanf("%d", &input);
		if (max < input) {
			max = input;
			num = i;
		}
	}
    
	printf("%d\n%d\n", max, num);
	return 0;
}

File: 백준/Bronze/2562. 최댓값/최댓값.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf8').trim().split(/\s+/).map(Number);

let max = input[0];
let index = 1;

for (let i = 1; i < input.length; i++) {
  if (input[i] > max) {
    max = input[i];
    index = i + 1;
  }
}

console.log(max + "\n" + index);